### PR TITLE
Junk pr

### PR DESCRIPTION
**Pull request by JonKohler**
Alright, lets try this again, with even more junk and a proper golangci file

### CODE REVIEW

**Comment by JonKohler** [comment]
well, this is highly embarrassing. My apologies. This works for me when I do it on my project :( For example, see here: https://github.com/nutanix/terraform-provider-nutanix/pull/145

I'll open up an issue with the golangci team real quick.

**Comment by JonKohler** [comment]
I've filed an issue on the central tracker for golangCI here to see if we can figure out why this isn't flagging (anything) in this pile of garbage PR - https://github.com/golangci/golangci/issues/9

**Comment by casualjim** [comment]
as you can see golangci is now required

**Comment by JonKohler** [comment]
fantastic, thanks! Let's use this to drive to conclusion with upstream to figure out why its not doing what it should be doing

**Comment by JonKohler** [comment]
They've got an issue with their service for this repo, they are fixing it per: https://github.com/golangci/golangci/issues/9#issuecomment-397671873

File: cmd/swagger/swagger.go [hunk 1]
```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 package main
-
+package mainpackage mainpackage mainpackage mainpackage main
```
Comment (golangci) [inline cmd/swagger/swagger.go]: expected declaration, found 'package' (and 4 more errors)

File: cmd/swagger/swagger.go [hunk 2]
```diff
@@ -68,10 +82,10 @@ It aims to represent the contract of your API with a language agnostic descripti
 	if err != nil {
 		log.Fatal(err)
 	}
-
+s
```
Comment (golangci) [inline cmd/swagger/swagger.go]: undeclared name: s

File: cmd/swagger/swagger.go [hunk 3]
```diff
@@ -68,10 +82,10 @@ It aims to represent the contract of your API with a language agnostic descripti
...
 	_, err = parser.AddCommand("init", "initialize a spec document", "initialize a swagger spec document", &commands.InitCmd{})
 	if err != nil {
-		log.Fatal(err)
+		log.Fatal(err)sdf
```
Comment (golangci) [inline cmd/swagger/swagger.go]: undeclared name: log

**Comment by JonKohler** [comment]
As you guys can see, they fixed up the issue that I wrote about in the ticket on their system, and this is how they work for other PRs

happy hunting!

**Pull request closed without merging**